### ISSUE TITLE
Significance of a footer in the editor

### ISSUE THREAD

**Issue by gautamig54** [publiclab/plots2#5484]
I want to know the significance of the footer on the edit/write post editor. I feel it is redundant there and can be removed.

![Screen Shot 2019-04-15 at 4 25 02 PM](https://user-images.githubusercontent.com/35326753/56127634-3d07c600-5f9b-11e9-851e-5d28a783a38d.png)

**Comment by gautamig54** [comment]
@IshaGupta18 @CleverFool77 @divyabaid16 What do you think about this?

**Comment by IshaGupta18** [comment]
I think you might be right, although we can work on a combined footer
design for the entire site so that only the most relevant information is
put up there. What do you think?

**Comment by gautamig54** [comment]
Yes @IshaGupta18, redesigning a footer to accommodate relevant information is a great idea!
But I still think having a footer in editor is kind of redundant. What do you think?

**Comment by IshaGupta18** [comment]
Yes I think we could keep the disclaimer in a text and make a small panel
for publish and draft button. What do you think?

**Comment by gautamig54** [comment]
@IshaGupta18 I don't understand what you mean by disclaimer in a text. Can you please explain? Thanks!

**Comment by IshaGupta18** [comment]
Like the disclaimer text in the footer can be displayed like normal text
somewhere in the page. Does that make sense?

**Comment by gautamig54** [comment]
Yes, I understand! Thanks for explaining. I agree with you. 
Adding to what you said, the current editor has a history of edited notes, together with disclaimer text publish and draft button, we can add history as well. 
I would love to work on this issue!
Thanks @IshaGupta18 for your input!

**Comment by IshaGupta18** [comment]
Not a problem! Since it could involve a little variation, would you want to
split this up with me, if approved? What do you say?

**Comment by gautamig54** [comment]
Sure @IshaGupta18. Let's work on this together. 
@jywarren kindly give your reviews on this issue, so that we can start working on this. Thanks!!

**Comment by gautamig54** [comment]
@IshaGupta18 Can we start on this one?

**Comment by IshaGupta18** [comment]
Hey, actually I am starting with my end sems tomorrow, they are kinda
consecutive, could you wait just for a couple of days? Or you could split
it up and assign me a part and I'll execute it ASAP. What do you think?
Thanks a lot for your patience!

**Comment by gautamig54** [comment]
Yes, sure. I'll think about the design and split it up. All the best for you semester exams :)

**Comment by IshaGupta18** [comment]
Thanks a lot, just let me know of the design, we can brainstorm it, and
then do the code. What say?

**Comment by gautamig54** [comment]
Sure!!

**Comment by jywarren** [comment]
Hi all! Thank you! I think with the fixed-bottom footer that has the
Publish button, the site-wide standard footer does look confusing. We could
decide that this /post page is the only page on the site where we don't
display it, and wrap it in a conditional tied to the `params[:action]` and
`params[:controller]`, maybe? Or to the route, although i don't know how to
access that in the views.

I'd also love to work a bit on this issue, which presents even worse design
problems, especially on mobile:
https://github.com/publiclab/plots2/issues/4776

We'd love your help there too!!!

**Comment by gautamig54** [comment]
Hi @jywarren. I am working on this issue. I think its better if we remove the fixed-bottom footer for editor. 
We can then decide where to place the Publish button, history button and draft button.
What do you think?

**Comment by IshaGupta18** [comment]
@gautamig54 what is the progress on this issue? Can we think of something and split the work?
